web/app/components/new/template-form.hbs

```diff
@@ -0,0 +1,104 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!-- custom-template-add --}}
+{{#if this.docIsBeingCreated}}
```

Comment:
can these variables be renamed?

Reply:
yes . We will discuss together and change all the variables to their expected names